src/types/user/user.ts

```diff
@@ -14,3 +14,20 @@ export type UserBase = {
   phone_number: string;
   gender: (typeof GENDER_TYPES)[number]["id"];
 };
+
+export type UserCreateRequest = {
+  user_type: UserType;
+  gender: (typeof GENDER_TYPES)[number]["id"];
+  password: string;
+  geo_organization: string;
+  username: string;
+  first_name: string;
+  last_name: string;
+  email: string;
+  phone_number: string;
+  alt_phone_number?: string;
+  date_of_birth: string;
+  qualification?: string;
+  doctor_experience_commenced_on?: string;
+  doctor_medical_council_registration?: string;
```

Comment:
Hmm 🤔  I'm also adding something similar here: https://github.com/ohcnetwork/care_fe/pull/10027

cc: @rithviknishad 

DoB is going away btw.